packages/rich-text/src/component/index.js

```diff
@@ -988,6 +990,11 @@ class RichText extends Component {
 				onKeyUp={ this.onSelectionChange }
 				onMouseUp={ this.onSelectionChange }
 				onTouchEnd={ this.onSelectionChange }
+				role="textbox"
+				aria-multiline
+				aria-label={ placeholder }
```

Comment:
We're changing behavior here: Previously, any `ariaProps` passed from props would always take priority, but now they're applied before we apply the defaults.

Should we spread `...ariaProps` after these instead?